src/components/Tool/Appeal/InitialPage/AddAppealForm/AddAppealForm.tsx

```diff
@@ -0,0 +1,661 @@
+  },
+  {
+    name: i18n.t('Have "Send Appeals" set to No'),
+    value: ExclusionEnum.DoNotAskAppeals,
+  },
+];
+
+export const calculateGoal = (
+  initialGoal: number,
+  letterCost: number,
+  adminCost: number,
+): number => {
+  const adminPercent = 1 - Number(adminCost) / 100;
+
+  return (
+    Math.round(
+      ((Number(initialGoal) + Number(letterCost)) / adminPercent) * 100,
+    ) / 100
+  );
+};
+
+type BuildInclusionFilterProps = {
+  appealIncludes: object;
+  tags: Attributes['tags'];
+  statuses: Attributes['statuses'];
+};
+export const buildInclusionFilter = ({
+  appealIncludes,
+  tags,
+  statuses,
+}: BuildInclusionFilterProps): object => {
+  const defaultInclusionFilter = {
+    any_tags: true,
+  };
+
+  const inclusionFilter = removeObjectNulls({
+    ...defaultInclusionFilter,
+    tags: tags && tags.length ? tags.join(',') : null,
+    status:
+      statuses && statuses.length
+        ? statuses.map((status) => status.value).join(',')
+        : null,
+    ...appealIncludes,
+  });
+
+  return isEqual(inclusionFilter, defaultInclusionFilter)
+    ? {}
+    : inclusionFilter;
+};
+
+export const buildExclusionFilter = (
+  exclusions: Attributes['exclusions'],
+): object => {
+  if (!exclusions || !exclusions.length) {
+    return {};
+  }
+  const today = DateTime.local().toFormat('yyyy-MM-dd');
+  const threeMonthsAgo = DateTime.local()
+    .minus({ months: 3 })
+    .toFormat('yyyy-MM-dd');
+  const containsExclusion = (exclusionValue: ExclusionEnum) => {
+    return exclusions?.find((exclusion) => exclusion.value === exclusionValue);
+  };
+
+  return removeObjectNulls({
+    started_giving_range: containsExclusion(ExclusionEnum.JoinedTeam)
+      ? `${threeMonthsAgo}..${today}`
+      : null,
+    gave_more_than_pledged_range: containsExclusion(ExclusionEnum.SpecialGift)
+      ? `${threeMonthsAgo}..${today}`
+      : null,
+    pledge_amount_increased_range: containsExclusion(
+      ExclusionEnum.IncreasedGiving,
+    )
+      ? `${threeMonthsAgo}..${today}`
+      : null,
+    pledge_late_by: containsExclusion(ExclusionEnum.MissedGift)
+      ? '30_90'
+      : null,
+    no_appeals: containsExclusion(ExclusionEnum.DoNotAskAppeals) ? true : null,
+  });
+};
+
+const appealFormSchema = yup.object({
+  name: yup.string().required('Please enter a name'),
+  initialGoal: yup
+    .number()
+    .typeError(i18n.t('Initial Goal must be a valid number'))
+    .required(i18n.t('Initial Goal is required'))
+    .test(
+      i18n.t('Is positive?'),
+      i18n.t('Must use a positive number for Initial Goal'),
+      (value) => parseFloat(value as unknown as string) >= 0,
+    ),
+  letterCost: yup
+    .number()
+    .typeError(i18n.t('Letter Cost must be a valid number'))
+    .required(i18n.t('Letter Cost  is required'))
+    .test(
+      i18n.t('Is positive?'),
+      i18n.t('Must use a positive number for Letter Cost'),
+      (value) => parseFloat(value as unknown as string) >= 0,
+    ),
+  adminCost: yup
```

Comment:
This amount is a percentage, right? Can we rename the field to `adminPercentage` for clarity?